src/services/url.ts

```diff
@@ -5,17 +5,21 @@ import { core, BaseService } from '../core/service';
 import UrlBeautifier from '../core/url-beautifier';
 import { WINDOW } from '../core/utils';
 import StoreFront from '../storefront';
+import Utils from './urlUtils';
 
 export const STOREFRONT_APP_ID = 'GroupBy StoreFront';
 
 @core
 class UrlService extends BaseService<UrlService.Options> {
+  static getBasePath: Function = Utils.getBasePath;
```

Comment:
try to never use the `Function` type as it is extremely generic
One thing you can do here to declare the type dynamically:
```js
getBasePath: typeof Utils.getBasePath = Utils.getBasePath;
```